package.nls.json

```diff
@@ -40,5 +40,7 @@
     "azFunc.enableRemoteDebugging": "Enable remote debugging, an experimental feature that only supports Java-based Functions Apps.",
     "azFunc.deleteProxy": "Delete Proxy",
     "azFunc.pickProcessTimeoutDescription": "The timeout (in seconds) to be used when searching for the Azure Functions host process. Since a build is required every time you F5, you may need to adjust this based on how long your build takes.",
-    "azFunc.showFuncInstallationDescription": "Show a warning to install Azure Functions Core Tools CLI when you create a new project if the CLI is not installed."
+    "azFunc.showFuncInstallationDescription": "Show a warning to install Azure Functions Core Tools CLI when you create a new project if the CLI is not installed.",
+    "azFunc.backupTemplateRuntime-~1": "The v1 runtime template release to use if retrieving templates from cache and latest both result in failure.",
```

Comment:
I actually don't see the 'backup' behavior as the main driver for this setting. My main motivation is for users that don't want the latest templates. In other words, this setting should override the version in `tryGetLatestTemplateData`.

That behavior would actually serve as a more robust 'backup' anyways. Consider this scenario: The cliFeed continues to list all templates, but the 'index.js' file is missing for all JavaScript templates. With your current implementation, there's nothing the user can do since the `backupTemplateRuntime` only applies to the backup templates and those won't be used in this scenario. With my suggestion, they can actually change the setting to an older version that still has 'index.js' and it will be used.